app/views/submissions/_form.html.erb

```diff
@@ -63,8 +67,26 @@
 					<%= t('submission.no_manuscript_yet') %>
 				</span>
 			</div>
-	    <div class="col-12" id="userManuscript-informer">
-  	  </div>
+	    <div class="col-12" id="userManuscript-status">
+			<div class="col-12" id="file-size-correct" style="display:none"><i class="fas fa-check"></i> <%=t('submission.correct_file_size')%></div>
+			<div class="col-12" id="file-size-too-big" style="display:none"> <i class="fas fa-exclamation-triangle"></i> <%=t('submission.file_size_too_big')%></div>
+			<div class="col-12" id="file-size-way-too-big" style="display:none"><i class="fas fa-bomb"></i> <%=t('submission.file_size_way_too_big')%></div>
+			<div class="col-12" id="file-optimize" style="display:none">
+			<small><%=raw t("submission.optimization_help")%><%= helpdesk(t('submission.optimization_info'),true)%></small>
+			<div class="col-12">
+			<button id="userManuscript-optimize-btn" class="btn btn-sm btn-outline-secondary"><%= t('submission.optimize') %></button>
+			</div>
+			<div class="col-1"></div>
+			<div class="col-11"><b id="log-btn">Log (show)</b>
+			<div id="userManuscript-optimize-log" style="display:none"></div>
+			</div>
+			</div>
+			<div class="col-12"><input type="checkbox" id="file-permission-checkbox"/> <%= t('submission.assure_third_party') %></div>
+			<div class="col-12">
+			<div class="col-3"><button id="userManuscript-uploadButton-call" class="btn btn-sm btn-outline-secondary" disabled=true><%= t('basics.upload') %></button></div>
```

Comment:
Same as above: I would suggest `btn-primary` instead of `btn-outline-secondary`.

Also, `basics.upload` is not yet defined in the locale files (I would probably rather go for `buttons.upload`, but that is actually equal to me).